
 This patch activates the Close button in the tray monitor window.
 It fixes bug #986.

 Apply it to Bacula version 2.2.5 with:

 cd <bacula-source>
 patch -p0 <2.2.5-tray-monitor.patch
 make
 ...
 make install

Index: src/tray-monitor/tray-monitor.c
===================================================================
--- src/tray-monitor/tray-monitor.c	(revision 5717)
+++ src/tray-monitor/tray-monitor.c	(working copy)
@@ -478,7 +478,7 @@
    gtk_box_pack_start(GTK_BOX(hbox), button, TRUE, FALSE, 0);
 
    button = new_image_button("gtk-close", _("Close"));
-// g_signal_connect_swapped(G_OBJECT(button), "clicked", G_CALLBACK(gtk_widget_hide), G_OBJECT(window));
+   g_signal_connect_swapped(G_OBJECT(button), "clicked", G_CALLBACK(gtk_widget_hide), G_OBJECT(window));
    gtk_box_pack_start(GTK_BOX(hbox), button, TRUE, FALSE, 0);
 
    gtk_box_pack_start(GTK_BOX(vbox), hbox, FALSE, FALSE, 0);
